### PR TITLE
Feature/info button

### PR DESCRIPTION
**Pull request by Oxydeas**
**Basic retrieval of info from current house**
Currently in a separate activity and unformatted.

### CODE REVIEW

**Review by sansive — COMMENTED** [review]
Maybe it is best if you use the current string HOUSEHOLD to send the name of the house to the info activity and then retrieve the data

File: app/src/main/res/layout/activity_info.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".InfoActivity">
+
+    <TextView
+        android:id="@+id/info_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="No info yet!"
```
Comment (aindreias) [inline app/src/main/res/layout/activity_info.xml:13]: Maybe extract the "No info yet!" string

**Review by aindreias — APPROVED** [review]
Thanks for the contribution, it's short but useful.

**Review by Defibrilator — APPROVED** [review]
Good, nothing much to say, an info layout is a future task.

**Comment by Oxydeas** [comment]
> Maybe it is best if you use the current string HOUSEHOLD to send the name of the house to the info activity and then retrieve the data

Yes, I probably will in a future task but for now I think it is fine. If I wanted to do that I would have to retrieve the database again in the info activity etc. which might not be ideal

File: app/src/main/res/layout/activity_info.xml (hunk 1 above)
Comment (Oxydeas) [reply app/src/main/res/layout/activity_info.xml:13]: Commit coming up - Thanks

**Pull request merged**